.github/workflows/postman.yml

```diff
@@ -0,0 +1,47 @@
+name: Postman Tests
+
+on:
+  pull_request:
+    branches: [ main, release/** ]
+  push:
+    branches: [ main, release/** ]
+    tags: [ v* ]
+
+jobs:
+  setup-local-hedera:
+    name: Acceptance Tests
```

Comment:
I wonder if this should have a separate name?

Reply:
Yes good, catch. Fixed it above missed this one

Reply:
Fixed